docs/notebooks/dispersion/dispersion_function.ipynb

```diff
@@ -84,7 +84,7 @@
    },
    "outputs": [],
    "source": [
-    "x = np.linspace(-1, 1, 1000)\n",
+    "x = np.linspace(-1, 1, 200)\n",
```

Comment:
The reason why I happened across `plasma_dispersion_function` in the first place was because in #2268 it said that this notebook was the slowest one to be processed in the documentation build.  I'm not sure why, but I'm reducing the resolutions to hopefully speed it up.